tests/vision/vision_conftest.py

```diff
@@ -45,10 +50,41 @@
            'coco_test_dataloader',
            'coco_test_visiondata',
            'two_tuples_dataloader',
-           'mnist_drifted_datasets'
+           'mnist_drifted_datasets',
+           'mock_trained_yolov5_object_detection',
+           'mock_trained_mnist',
            ]
 
 
+def _hash_image(image):
+    if isinstance(image, np.ndarray):
+        image = Image.fromarray(image)
+    elif isinstance(image, torch.Tensor):
+        image = Image.fromarray(image.cpu().numpy().squeeze())
+
+    image = image.resize((10, 10))
+    image = image.convert('L')
```

Comment:
what is L?

Reply:
Converting the image to a single-channel image